scripts/generate_genotype_df/generate_genotype_df.py

```diff
@@ -24,8 +25,15 @@ def query():
     mt = mt.filter_rows(hl.agg.sum(hl.is_missing(mt.GT)) > (n_samples * call_rate), keep=False)
     # filter out variants with MAF < 0.01
     mt = mt.filter_rows(mt.freq.AF[1] > 0.01)
-    tob_wgs_path = output_path('tob_wgs_maf01.parquet')
-    mt.rows().to_pandas()[['locus.contig','locus.position','alleles']].to_parquet(tob_wgs_path)
+    # select only locus and alleles, which are the keys, then convert to pandas
+    locus_alleles = mt.rows().select().to_pandas(flatten=True)
+    # expand locus to two columns and rename 
+    locus_alleles = locus_alleles.astype(str)
```

Comment:
Instead of converting `Locus` to a string and then pulling it apart using `split`, can you use the `contig` and `position` fields directly?
https://hail.is/docs/0.2/genetics/hail.genetics.Locus.html#hail.genetics.Locus

Reply:
I tried to do a matrix.locus.contig.collect_as_set() but the notebook ran out of memory!

Reply:
@MattWellie I've never seen `collect_as_set()` before. I just tried it on a small subset now. 

@lgruen The issue I'll have if I use `mt.locus.contig` or `mt.locus.position` directly is that it gets converted to a string expression rather than staying as a hail table (which has the `to_pandas` functionality). The best I could come up with instead is something like this:
`mt = mt.annotate_rows(contig = mt.locus.contig, position=mt.locus.position).key_rows_by('alleles')
locus_alleles = mt.rows().select('contig','position').to_pandas(flatten=True)
`

I can't select rows by `mt.locus.contig` or `mt.locus.position` directly, unfortunately

Reply:
Could you instead say:
```
t = mt.rows()
t = t.key_by(contig = t.locus.contig, position = t.locus.position)
t = t.select(t.alleles)
pd = t.to_pandas(flatten=True)
```
?

Reply:
that's a lot more readable. I've incorporated this version in. Thanks!